app/src/main/java/com/example/lutsak/bitmapimage/MainActivity.java

```diff
@@ -0,0 +1,85 @@
+package com.example.lutsak.bitmapimage;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.ImageView;
+import java.io.IOException;
+import java.net.URL;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final String TAG = MainActivity.class.getSimpleName();
+
+    public static final String IMAGE_URL = "https://www.google.com/images/srpr/logo11w.png";
+
+    private ImageView mImageView;
+    private DownLoadImageTask mLoadImageTask;
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        Log.d(TAG, ">> onCreate");
+
+        mImageView = findViewById(R.id.image_view);
+
+        /*
+          Download the logo from online and set it as
+          ImageView image programmatically.
+         */
+        mLoadImageTask = new DownLoadImageTask(mImageView);
```

Comment:
Looks like after screen rotation you will start fetching the Bitmap once again.
Please check this and fix

Reply:
I'm going to fix it as soon as possible.